transport/grpc/options.go

```diff
@@ -128,6 +129,20 @@ func ClientTLS() TransportOption {
 	}
 }
 
+// ClientTLSConfig provides a tls.Config for the GRPC client
```

Comment:
Nit: period at end of sentence, and match the other options by saying what the default behavior is (no TLS). Also mention this overrides the `ClientTLS()` option.